packages/perennial/deploy/005_deploy_multiinvoker.ts

```diff
@@ -35,45 +35,45 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const usdc = await IERC20__factory.connect(usdcAddress, deployerSigner)
 
   const multiInvokerImpl: Deployment = await deploy('MultiInvoker_Impl', {
-    contract: 'MultiInvoker',
+    contract: 'MultiInvokerRollup',
```

Comment:
We should use the `isArbitrum`/`isBase` (https://github.com/equilibria-xyz/perennial-mono/blob/master/packages/common/testutil/network.ts#L72) helpers to check which network we're on and then deploy the correct contract based on that 

We do something similar in the Core deployment scripts